src/repo/feature-details.repo.js

```diff
@@ -37,6 +37,7 @@ function formatDataIntoLinks(data) {
 
   const buildLinks = compose(
     map(formatLink),
+    val => Array.isArray(val) ? val : [val],
```

Comment:
explain what you are doing here?

Reply:
Basically an `ensureArray` operation. The whole pipeline was built to expect arrays (`map`, `v-for`) and this new layer threw me a curveball by not returning an array from the WPS. Hence I'm wrapping it in an array if applicable.